js/views/drawerView.js

```diff
@@ -72,8 +63,24 @@ class DrawerView extends Backbone.View {
     this.checkIfDrawerIsAvailable();
   }
 
-  openCustomView(view, hasBackButton) {
+  setDrawerPosition(position) {
+    this.$el
+      .removeClass(`is-position-${this.drawerPosition}`)
+      .addClass(`is-position-${position}`);
+    this.drawerPosition = position;
+    this.drawerAnimationDir = (position === 'auto') ? ((Adapt.config.get('_defaultDirection') === 'rtl') ? 'left' : 'right') : position;
+  }
+
+  getInversePosition(position) {
```

Comment:
This function looks like an afterthought.

Could you move all the position stuff out out openCustomView, change setDrawerPosition to setUpDrawerPosition and either move the code body of getInversePosition into setUpDrawerPosition or rename `getInversePosition` to `get position()` including the `_globalDrawerPosition` behaviour inside it?

Reply:
Have moved all the positional code into `setDrawerPosition` to remove `getInversePosition`. Haven't used a getter for position as we need to determine if a position parameter has been set by a plugin.